ci/properties/objective-c-xcode.properties.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "Build Xcode project",
```

Comment:
Will this template build any xcode project and not just Objective-c language ones? Is there a case where it can't build objective-c code?

Reply:
@aparna-ravindra my question was other way round. Is it possible that an Objective-c code written using xcode does not build using the above command? If so, then adding `objective-c` in categories might be questionable.

Reply:
No, the command will build all objective-c projects developed using xcode.

Reply:
got it. Thanks